addons/website/static/src/js/menu/content.js

```diff
@@ -22,6 +22,8 @@ var PagePropertiesDialog = weWidgets.Dialog.extend({
         'keyup input#page_name': '_onNameChanged',
         'keyup input#page_url': '_onUrlChanged',
         'change input#create_redirect': '_onCreateRedirectChanged',
+        'click input#visibility_password': '_onPasswordClicked',
```

Comment:
`focus` might be a better even here in case the user tabs from field to field

Reply:
imo, it is better with only on click, since when you switch from keyboard, you risk to enter in the field just to go to the next one, you will reset and mark it as dirt for nothing. And default behaviour to select all on foux, is enough to hope that user will replace all the password and not append a few char after the ****** 'placeholder' ;) no?

Reply:
makes sense.